src/server/shared/SharedDefines.h

```diff
@@ -2724,7 +2724,7 @@ enum CreatureTypeFlags
     CREATURE_TYPE_FLAG_BOSS_MOB                             = 0x00000004,   // Changes creature's visible level to "??" in the creature's portrait - Immune Knockback.
     CREATURE_TYPE_FLAG_DO_NOT_PLAY_WOUND_PARRY_ANIMATION    = 0x00000008,
     CREATURE_TYPE_FLAG_HIDE_FACTION_TOOLTIP                 = 0x00000010,
-    CREATURE_TYPE_FLAG_UNK5                                 = 0x00000020,   // Sound related
+    CREATURE_TYPE_FLAG_MORE_AUDIBLE                         = 0x00000020,   // Sound related
```

Comment:
What is meant with "more audible"? "Sound related" doesn't describe why it was renamed like that.

Reply:
"More audible" looks like a description meant to cover both "louder" and "making more sounds" compared to similar creatures. (I am unfamiliar with the scenario.)

Reply:
These names are based on the leaked enum db2 file from the tbc classic client so that comment is the thing that you should ignore for the time being. There is lots of juicy renaming incoming with official names

Reply:
Thanks. Nice. Good to know. 😄